docs/modules/ROOT/pages/tls.adoc

```diff
@@ -74,3 +74,5 @@ include::ROOT:example$/hazelcast-tls.yaml[]
 ----
 include::ROOT:example$/management-center-tls.yaml[]
 ----
+
+NOTE: To update TLS certificates, it is required to delete and recreate the Hazelcast/Management Center custom resource.
```

Comment:
```suggestion
NOTE: To update TLS certificates, you need to delete and recreate the Hazelcast/Management Center custom resource.
```
Is it either the Hazelcast or MC resource you need to delete or both? The / is ambiguous here...

Reply:
TLS certificates are applicable for both CRs. So whichever you created you should delete it and recreate.

Reply:
What do you think of update suggestion then?

Reply:
I think this should be enough: `NOTE: To update TLS certificates, you need to delete and recreate the appropriate custom resource`, WDYT?

Reply:
Yep, ship it!